lib/postprocess/image.js

```diff
@@ -100,13 +100,10 @@ module.exports = async files => {

```

Comment:
@sebastian-schlecht this seems very unsexy, even if I insert breaks to keep the line length down - is there an alternative to storing the string on disk but rather in the code?